pkg/registry/core/node/strategy.go

```diff
@@ -106,6 +106,10 @@ func dropDisabledFields(node *api.Node, oldNode *api.Node) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.RecursiveReadOnlyMounts) {
 		node.Status.RuntimeHandlers = nil
 	}
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.SupplementalGroupsPolicy) {
```

Comment:
This logic is supposed to follow the pattern of

```
if !gate-enabled && !in-use(oldObj) {
    wipe it
}
```

This allows for users to set the field, then disable the gate, and not break.  It's sort of an edge case, but it's an established pattern.  https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md#using-a-feature-gate-in-code

Reply:
I will note that the one right above this is also wrong.

Reply:
Thanks for both pointing out an edge case and fixing docs!

fixed in https://github.com/kubernetes/kubernetes/pull/125470/commits/2fe1fdaada2e7cd28f614ccc9a5f6e1c49a9c187